src/codegen/doc/mod.rs

```diff
@@ -21,6 +21,16 @@ use stripper_lib::{write_file_name, write_item_doc, Type as SType, TypeStruct};
 
 mod format;
 
+// A list of C parameters that are not used directly by the Rust bindings
+const IGNORED_C_FN_PARAMS: [&str; 6] = [
+    "user_data",
+    "user_destroy",
+    "destroy_func",
+    "dnotify",
+    "destroy",
+    "user_data_free_func",
```

Comment:
You should probably also check the types or not?

Reply:
And maybe add a configuration (per function etc) to override this?

Reply:
Not sure what to override exactly ?

Reply:
In general, this list should only contain parameters that are known per conventions to be not useful at all for the rust bindings (form a user perspective) if something on the list proves to cause issues, it should be special cases like `data` a bit below where it can mean `user_data` or something else.

Reply:
Hiding other parameters from the documentation, for example length parameters of arrays or destroy notifies with non-default names.

Reply:
Ah I see, the array length parameters are already hidden based on the introspection data (the `length` annotation) so if there's some case, it would need an annotation fix instead :)

Reply:
I will have a look later today in order to add a config for :)

Reply:
Actually, wasn't that hard so I implemented it already :-)

Reply:
This was actually a good idea as I found two use cases for it on gtk4-rs where array length annotation isn't properly picked for TreeStore/ListStore